From 4a4a601b6af71d4a2ba544b7915d114da99e6e42 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 24 Sep 2015 11:25:33 +0800
Subject: [PATCH] MLK-11614 ARM: dts: imx6qp-sabresd: Enable prefetch resolve
 gasket(PRG) engines

This patch enables the two prefetch resolve gasket(PRG) engines on the
imx6qp-sabresd platform.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi |  2 +-
 arch/arm/boot/dts/imx6qp-sabresd.dts   | 10 ++++++++++
 2 files changed, 11 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 0745b18..df76622 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -55,7 +55,7 @@
 		};
 	};
 
-	memory {
+	memory: memory {
 		reg = <0x10000000 0x40000000>;
 	};
 
diff --git a/arch/arm/boot/dts/imx6qp-sabresd.dts b/arch/arm/boot/dts/imx6qp-sabresd.dts
index 7040ecd..8fe63a5 100644
--- a/arch/arm/boot/dts/imx6qp-sabresd.dts
+++ b/arch/arm/boot/dts/imx6qp-sabresd.dts
@@ -28,3 +28,13 @@
 &pre4 {
 	status = "okay";
 };
+
+&prg1 {
+	memory-region = <&memory>;
+	status = "okay";
+};
+
+&prg2 {
+	memory-region = <&memory>;
+	status = "okay";
+};
-- 
1.8.0

